src/hotspot/share/utilities/vmError.hpp

```diff
@@ -188,8 +188,11 @@ class VMError : public AllStatic {
   DEBUG_ONLY(static void controlled_crash(int how);)
 
   // Non-null address guaranteed to generate a SEGV mapping error on read, for test purposes.
-  static constexpr intptr_t segfault_address = (1 * K) AIX_ONLY(+ (4 * K));
-
+#ifdef AIX
+  static constexpr intptr_t segfault_address = -1;
+#else
+  static constexpr intptr_t segfault_address = (1*K);
+#endif //AIX
```

Comment:
Nit, could you please use 
```
static constexpr intptr_t segfault_address = AIX_ONLY(-1) NOT_AIX(1*K);
```
instead? Its a bit tighter coding.

Thanks!

Reply:
Okay. Thank you